rpm/generic/zfs-dkms.spec.in

```diff
@@ -27,6 +27,7 @@ Requires:       gcc, make, perl
 Requires:       kernel-devel
 %endif
 Provides:       %{module}-kmod = %{version}
+AutoReqProv:    no
```

Comment:
The resulting rpm misses these auto-dependencies:
```
auto: /bin/bash
auto: /bin/sed
auto: /bin/sh
auto: /usr/bin/env
auto: /usr/bin/python
```
`/bin/sh` and `/usr/bin/env` should be safe to assume "already installed", probably.

I tried to remove the other three (`bash`, `sed` and `python`) on my CentOS7 but they are all needed by `yum`, it is probably safe to assume any RHEL-based will always have those installed and we don't need to manually "require" them here.

OTOH on Debian8 i can safely uninstall `python`: this most probably comes from the ZTS test runner and `arcstat.py`/`arc_summary.py`/`dbufstat.py`, so it's ok to drop from the DKMS package too.

That being said, LGTM.

Reply:
Yes, I considered dropping some of these for the reasons you mentioned by opted not to out of an abundance of caution.  Worst case we pull in some additional dependencies which are very likely already installed.  If we want to strip down the dependencies further let's do it in a separate PR.